src/actions/types.js

```diff
@@ -7,8 +7,14 @@ export const SOCIAL_LOGIN_SUCCESS = 'SOCIAL_LOGIN_SUCCESS';
 export const GET_PROFILE_PAYLOAD = 'GET_PROFILE_PAYLOAD';
 export const GET_PROFILE_INITIATED = 'GET_PROFILE_INITIATED';
 
-export const GET_ALL_ARTICLES_SUCCESS = 'GET_ALL_ARTICLES_SUCCESS';
 export const CREATE_ARTICLE_SUCCESS = 'CREATE_ARTICLE_SUCCESS';
 export const CREATE_ARTICLE_INITIATED = 'CREATE_ARTICLE_INITIATED';
 export const CREATE_ARTICLE_ERROR = 'CREATE_ARTICLE_ERROR';
 export const LOGOUT_USER = 'LOGOUT_USER';
+export const SEND_RESET_LINK_SUCCESS = 'SEND_RESET_LINK_SUCCESS';
```

Comment:
These exports are related to the resetting password feature.
Please remove them.